src/fides/api/task/deprecated_graph_task.py

```diff
@@ -109,54 +109,54 @@ def run_access_request_deprecated(
 ) -> Dict[str, List[Row]]:
     """Deprecated: Run the access request sequentially in-memory using Dask"""
     traversal: Traversal = Traversal(graph, identity)
-    resources = TaskResources(
+    with TaskResources(
         privacy_request, policy, connection_configs, EMPTY_REQUEST_TASK, session
```

Comment:
This was the DSR 2.0 regression I caused, using this a context manager had functionality to close open connections